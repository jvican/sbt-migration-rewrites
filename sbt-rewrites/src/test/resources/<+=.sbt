lazy val seqKey = taskKey[Seq[File]]("Target to be reassigned.")
lazy val seed = taskKey[File]("Seed.")

seqKey := Seq.empty
seed in ThisBuild := new File(".")

seqKey <+= seed

seqKey <+= (seed)

seqKey <+= (seed in ThisBuild)

seqKey <+= Def.task(seed.value)

seqKey.<+=[File](Def.task(seed.value))

seqKey <+= Def.task {
  println("Executing task.")
  (seed).value
}
