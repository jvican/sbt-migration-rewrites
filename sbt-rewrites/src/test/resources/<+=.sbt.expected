lazy val seqKey = taskKey[Seq[File]]("Target to be reassigned.")
lazy val seed = taskKey[File]("Seed.")

seqKey := Seq.empty
seed in ThisBuild := new File(".")

seqKey += seed.value

seqKey += (seed).value

seqKey += (seed in ThisBuild).value

seqKey += Def.task(seed.value).value

seqKey.+=[File](Def.task(seed.value).value)

seqKey += (Def.task {
  println("Executing task.")
  (seed).value
}).value
